README.md

```diff
@@ -1 +1,16 @@
-# PORTFOLIO-MOBILE
\ No newline at end of file
+PORTFOLIO-MOBILE
+Build a mobile portfolio webpage by following figma design.
+
+Built With
+HTML AND CSS
+
+
+Getting Started
+create a repository on github and clone it to my local environment. 
+Set a HTML and CSS linters in this project to guide me with errors.
+This project is built on an example of a portfolio page on figma.
+The design on figma is closely replicated in this project.
+
+👤 Aneal Laryea
+
+GitHub: @aneallaryea100
```

Comment:
I noticed that the previous reviewers requested to use of the proper Readme template, to make your project professional. So kindly use the good template [here](https://github.com/microverseinc/readme-template/blob/master/README.md)

At the end, your readme must look like:
<img width="598" alt="image" src="https://user-images.githubusercontent.com/7243132/170590079-b7085a92-4a16-42f2-b775-2063042c6e3c.png">